server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -489,19 +492,27 @@ fn create_filtered_outgoing_sync_query(
     // Loop through all the Sync tables and add them to the query if they have the right sync style
 
     // Central Records
-
     let central_sync_table_names: Vec<ChangelogTableName> = ChangelogTableName::iter()
         .filter(|table| matches!(table.sync_style(), ChangeLogSyncStyle::Central))
         .collect();
 
     // Remote Records
     let remote_sync_table_names: Vec<ChangelogTableName> = ChangelogTableName::iter()
         .filter(|table| {
-            matches!(table.sync_style(), ChangeLogSyncStyle::Remote)
-                || matches!(table.sync_style(), ChangeLogSyncStyle::RemoteToCentral)
+            matches!(
+                table.sync_style(),
+                ChangeLogSyncStyle::Remote
+                    | ChangeLogSyncStyle::RemoteAndCentral
+                    | ChangeLogSyncStyle::RemoteToCentral
```

Comment:
I'm now questioning if `RemoteToCentral` should be here, even though it's not related to this PR. Won't that mean initialisation will bring back the `RemoteToCentral` records when they shouldn't need to be on the remote site any more?

Reply:
Oo yeah, just tested this and yes, they are getting sent back to remote!

Reply:
@andreievg do you want to remove in this PR or can make sep issue

Reply:
Thanks, changed here: 8cd7bd76df37606588d0ff437dda13f6d2fc6ae1

I had to modify translation slightly for pull and push to work, ran all tests, work fine, and logically look good, but I don't want to merge unless someone has a quick look at that commit